comp/process/gpusubscriber/def/component.go

```diff
@@ -0,0 +1,14 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package gpusubscriber implements a component to subscribe to WorkloadMeta GPU events in the Agent.
+package gpusubscriber
+
+// team: container-intake
+
+// Component is the component type.
+type Component interface {
+	GetGPUTags() map[int32][]string
```

Comment:
[minor] Let's add a comment to describe the method here. Especially what the map returns (pid -> []tags).